examples/codeexecution/container/README.md

```
@@ -0,0 +1,190 @@
+
+```go
+executor, err := container.New(
+    container.WithBindMount("/host/inputs", "/data/inputs", "ro"),
+)
+```
+
+## Example Output
+
+When you run the example, you might see output like:
+
+    Creating LLMAgent with Container code executor:
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)